test/test_lib.py

```diff
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
```

Comment:
why?

Reply:
Because it's not cross-platform at all. See: https://docs.python.org/3/using/windows.html#shebang-lines

`#! python3` is more "inclusive"

Reply:
Not a big deal to me. I'm just used to work on Debian or Ubuntu and then test on Windows. Not sure which one is the best here.

Reply:
I t seems we can cumulate both:

```python
#!/usr/bin/env python3
```

Why not?

Reply:
Refs:

- https://stackoverflow.com/a/64061384/2556577
- chatgp 40 (to be honest)

Reply:
I've made the change.